runelite-client/src/main/java/net/runelite/client/plugins/diaryprogress/DiaryProgressConfig.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2017, oplosthee <https://github.com/oplosthee>
```

Comment:
We're in 2018

Reply:
You're writing runescape-related content and saying we're in 2018